portfolio/src/main/webapp/index.html

```diff
@@ -25,84 +26,169 @@
       integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
       crossorigin="anonymous"
     ></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="task.js"></script>
     <script src="calendar.js"></script>
-
-    <!-- Files for Jasmine, Unit testing tool for Javascript -->
-    <link rel="stylesheet" type="text/css" href="jasmine/lib/jasmine-3.5.0/jasmine.css" />
-    <script src="jasmine/lib/jasmine-3.5.0/jasmine.js"></script>
-    <script src="jasmine/lib/jasmine-3.5.0/jasmine-html.js"></script>
-    <script src="jasmine/lib/jasmine-3.5.0/boot.js"></script>
+    <script src="request.js"></script>
   </head>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <a class="navbar-brand" href="#">Task Scheduler</a>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav mr-auto">
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Scheduling Preferences
+              </a>
+              <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                <a>Working Hours:</a>
+                <form class="px-4 py-3">
+                  <input type="time" id="working-hour-start" name="working-hour-start" class="form-control" value="09:00" />
+                  <a>to</a>
+                  <input type="time" id="working-hour-end" name="working-hour-end" class="form-control" value="17:00" />
+                </form>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <div class="row">
+        <div id="new-task-wrapper" class="col-md-12 col-lg-6 col-xl-3">
+<!--          <h2>What are your working hours for today?</h2>
+          <span>
+            <input type="time" id="working-hour-start" name="working-hour-start" value="09:00" />
+            to
+            <input type="time" id="working-hour-end" name="working-hour-end" value="17:00" />
+          </span>
+-->
+          <h2>Add the tasks you would like to schedule for today</h2>
+          <form id="new-task-form">
+            <div class="form-group">
+              <label for="new-task-name">Task name:</label>
+              <input type="text" id="new-task-name" class="form-control" name="new-task-name" placeholder="New Task"/>
+            </div>
 
-  <body>
```

Comment:
It seems the body opening tag was deleted, but at the bottom we still have the closing tag it. Either add this one back or delete the other one.

Reply:
Done.